RevenueCatUI/CustomerCenter/URLUtilities.swift

```diff
@@ -27,17 +27,29 @@ enum URLUtilities {
     @available(tvOS, unavailable)
     @available(watchOS, unavailable)
     static func createMailURLIfPossible(email: String, subject: String, body: String) -> URL? {
-        let encodedSubject = subject.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) ?? ""
-        let encodedBody = body.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) ?? ""
+        guard !email.isEmpty else { return nil }
 
-        let urlString = "mailto:\(email)?subject=\(encodedSubject)&body=\(encodedBody)"
+        var components = URLComponents()
+        components.scheme = "mailto"
+        components.path = email
 
-        if let url = URL(string: urlString),
-           UIApplication.shared.canOpenURL(url) {
-            return url
+        var queryItems: [URLQueryItem] = []
+
+        if !subject.isEmpty {
```

Comment:
Is there an easy way to check whether a string is blank? Couldn't find anything built-in so far. If not, I think `isEmpty` is fine.

Reply:
I think this is the easy way! Does Kotlin have better syntax or functionality?